com.sap.cloud.lm.sl.cf.persistence/src/main/resources/com/sap/cloud/lm/sl/cf/persistence/db/changelog/db-changelog-alter_process_log_alter_content_column_to_bytea_cf.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    logicalFilePath="com/sap/cloud/lm/sl/persistence/db/changelog/db-changelog-add_trigger_for_lm_sl_persistance_file_process_log_cf.xml"
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="i328376" id="alter_process_log_alter_content_column_to_bytea_cf">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="postgresql" />
+        </preConditions>
+        <delete tableName="process_log" />
+        <sql>
+            DROP TRIGGER IF EXISTS delete_from_system_table_process_log on process_log;
```

Comment:
What about delete_from_system_table_lm_sl_persistance_file?

Reply:
lm_sl_persistance_file still uses blob and will still need to have the trigger to ensure proper cleanup of blobs on delete.